ckanext/gla/plugin.py

```diff
@@ -33,14 +38,84 @@ def get_auth_functions(self):
     def before_dataset_search(self, search_params):
         # Include showcases *and* datasets in the search results:
         # We only want Showcases to show up when there is a search query
-        if search_params.get("q", "") != "":
-            fq = search_params.get("fq", "")
-            search_params.update(
```

Comment:
Is this removed as part of the return to ckan search defaults?

Reply:
Yep, this was removed to rely as much as possible on default CKAN behaviour. Once we've carried out more tests in terms of search quality, we can be intentional about what we add with a detailed explanations of if and why we diverge from default behaviour. Rick's next PR has more changes on this.